web/modules/custom/joinup_subscription/joinup_subscription.module

```diff
@@ -111,8 +131,9 @@ function joinup_subscription_mail_alter(array &$message): void {
     $notifier = $message['params']['notifier'] ?? NULL;
     if ($notifier instanceof MessageNotifierInterface) {
       $interval = $notifier->getPluginDefinition()['title'];
-      $message['subject'] = t('Joinup: @interval digest message', [
+      $message['subject'] = t('Joinup: @interval:bundle_label digest message', [
         '@interval' => $interval,
+        ':bundle_label' => ' ' . $notifier->bundle_label ?? '',
```

Comment:
This is suspicious, what is this bundle label that is appended to the title? It looks like this change is intended to differentiate between different digest messages that are sent. Like a "collection digest message" and a "solution digest message".

This is not what we want. There should only be 1 digest sent to a user (for example the "Joinup weekly digest message") and this should include all messages, regardless of the bundle.